authentication/models.py

```diff
@@ -238,3 +239,17 @@ class TwitterConnection(BaseThirdPartyConnection):
         max_length=255, unique=True, blank=True, null=True
     )
     driver = TwitterDriver()
+
+
+class ENSSaveError(Exception):
+    pass
```

Comment:
when do you raise this ERROR?

Reply:
in ensconncectionview when creating the connection i raise this error

Reply:
I can't see that, you catch this ERROR but never raise it.

Reply:
yes you are right, so should i delete it because i think it is not needed, i have added it to be like other connections

Reply:
yes, I think you must delete it.